acceptance/ui/features/services.feature

```diff
@@ -29,4 +29,183 @@ Feature: Service Management
       And I should see "Public Endpoints"
       And VHost "table://services/childServiceWithVHost/vhostName" should exist
       And Public Port "table://services/childServiceWithVHost/publicPortNumber" should exist
-      And I should see "table://services/childServiceWithVHost/configFile"
\ No newline at end of file
+      And I should see "table://services/childServiceWithVHost/configFile"
+
+  @services_endpoint
+  Scenario: Public Endpoint dialog - check all labels
```

Comment:
I didn't see these changes to address this comment in 7c4fe6d528ba5f58b793b5a9a11a62130b94c2a4.  Did I miss something?